variables.tf

```diff
@@ -51,6 +35,28 @@ variable "ranger_database_name" {
   default     = "ranger"
 }
 
+variable "db_master_username" {
+  description = "Aurora cluster MySQL master user name."
+  type        = "string"
+  default     = "ranger"
+}
+
+variable "db_audit_username" {
+  description = "Ranger DB Audit user name."
+  type        = "string"
+  default     = "rangerlogger"
+}
+
+variable "ldap_secret_name" {
+  description = "Active directory LDAP bind DN SecretsManager secret name."
+  type        = "string"
+}
+
+variable "ldap_ca_cert" {
+  description = "Base64 encoded Certificate Authority bundle to validate LDAPS connections."
+  type        = "string"
+}
+
```

Comment:
So the intent is that these 2 new LDAP vars are essentially "breaking changes" in the sense that any TF app currently using `apiary-authorization 1.0.0` will have to change to add the LDAP certificate and LDAP secret name.  Should maybe be called out in the CHANGELOG or README specifically.

Reply:
updated CHANGELOG and README